### DISCUSSION TITLE
PowerShell 7.4.2 Available as a SNAP?

### DISCUSSION THREAD

**Discussion by SupSkiFun** [PowerShell/PowerShell#21494] [General]
Hi.  Any ETA when **PowerShell 7.4.2**  will be available as a SNAP?  As of UTC Wednesday, April 17, 2024 6:32:19 PM, it is still not listed at https://snapcraft.io/powershell.

**Comment by kilasuit** [comment]
@SupSkiFun - Seems it was published on the 3rd May 2024
